src/Mutator/Sort/Spaceship.php

```diff
@@ -74,6 +74,65 @@ public function mutate(Node $node): iterable
 
     public function canMutate(Node $node): bool
     {
-        return $node instanceof Node\Expr\BinaryOp\Spaceship;
+        if (!$node instanceof Node\Expr\BinaryOp\Spaceship) {
+            return false;
+        }
+
+        if ($this->isCompareWithZero($node)) {
+            return false;
+        }
+
+        return true;
+    }
+
+    private function isCompareWithZero(Node\Expr\BinaryOp\Spaceship $node): bool
+    {
+        $parentAttribute = $node->getAttribute('parent');
+
+        if ($parentAttribute instanceof Node\Expr\BinaryOp\Identical) {
+            if ($parentAttribute->right instanceof Node\Scalar\LNumber && $parentAttribute->right->value === 0) {
+                return true;
+            }
+
+            if ($parentAttribute->left instanceof Node\Scalar\LNumber && $parentAttribute->left->value === 0) {
+                return true;
+            }
```

Comment:
maybe you can return `false` here already?

What do you think of extracting the content of each `if()` in a private method for better readability?